lib/game.rb

```diff
@@ -0,0 +1,50 @@
+class Game
+  attr_reader :player1, :player2
+
+  # This is called with Game.create(player1, player2) where paramaters are player objects
+
+  def self.create(player_1, player_2)
+    @game = Game.new(player_1, player_2)
+  end
+
+  # that creates @game as an instance of the Class Game
+  # As the @game object exists at the same level the below method can see and return it.
+  # Therefore we can use it in all our routes by calling @game = Game.instance
+
```

Comment:
I like how you left comments in to explain your code to someone else/yourself in the future.